test/e2e/test_job_user.py

```diff
@@ -9,48 +9,21 @@
 import botocore.exceptions
 import pytest
 import logging
+import os
 
 from deadline_test_fixtures import Job, TaskStatus, PosixSessionUser, DeadlineClient
 
+
 LOG = logging.getLogger(__name__)
 
 
 @pytest.mark.usefixtures("session_worker")
+@pytest.mark.skipif(
+    os.environ["OPERATING_SYSTEM"] == "windows",
+    reason="Linux specific test",
+)
 @pytest.mark.parametrize("operating_system", ["linux"], indirect=True)
 class TestJobSubmission:
```

Comment:
nit: can we rename this to `TestJobUser` now that we are left with the one job user test?